modules/tinymce/src/core/main/ts/delete/DeleteUtils.ts

```diff
@@ -36,16 +36,9 @@ const getParentBlock = (rootNode: SugarElement<Node>, elm: SugarElement<Node>):
   }
 };
 
-const placeCaretInEmptyBody = (editor: Editor): void => {
-  const body = editor.getBody();
-  const node = body.firstChild && editor.dom.isBlock(body.firstChild) ? body.firstChild : body;
-  editor.selection.setCursorLocation(node, 0);
-};
-
-const paddEmptyBody = (editor: Editor): void => {
+const paddEmptyBody = (editor: Editor, moveSelection: boolean = true): void => {
   if (editor.dom.isEmpty(editor.getBody())) {
-    editor.setContent('');
-    placeCaretInEmptyBody(editor);
```

Comment:
Removed this, as it was unnecessary since `setContent` does set the cursor position nowadays, plus I believe this wouldn't have been the correct behaviour in some edge cases anyways.